proposals/lip-0062.md

```diff
@@ -60,6 +70,23 @@ Here, <code>[Sign](https://tools.ietf.org/html/rfc8032#section-5.1.6)</code> and
 
 The `signEd25519` and `verifyEd25519` functions defined in [LIP 0037](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0037.md#signing-and-verifying-with-ed25519) are superseded by the functions defined in this LIP.
 
+#### Signing Non-Protocol Messages with Ed25519
+
+A non-protocol related message `message` can be signed by the function `signNonProtocolEd25519`. The resulting signature can be verified by `verifyNonProtocolEd25519`. This scheme should be used for the _sign message_ feature in Lisk Desktop. Note that the tag `MESSAGE_TAG_NON_PROTOCOL_MESSAGE` is defined in [LIP 0037](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0037.md#message-tags-1).
+
+```python
+def signNonProtocolEd25519(sk: bytes, message: bytes) -> bytes:
```

Comment:
maybe not necessary to align, but for the SDK we have functions like
```
signMessageWithPrivateKey
verifyMessageWithPublicKey
```
so and it's used like `ed.signMessageWithPrivateKey`. I would prefer to keep those names

Reply:
I see the reason to keep the names in the SDK and it's fine for me. Would be good to haven some comments then. I'm also open to new names for the LIP as I'm not satisfied with them as well. But I didn't have a better idea. But I would like to avoid `signMessageWithPrivateKey` as it is really not descriptive.

Reply:
An alternative could be `signOffchainDataEd25519`.